src/components/shared/CallNodeContextMenu.css

```diff
@@ -1,34 +1,26 @@
-.callNodeContextMenuIcon {
-  position: relative;
-  top: 2px;
-  display: inline-block;
-  width: 16px;
-  height: 16px;
-  margin-inline-end: 8px;
-  pointer-events: auto;
-}
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-.react-contextmenu-item--selected > .callNodeContextMenuIcon,
-.react-contextmenu-item:hover .callNodeContextMenuIcon,
-.react-contextmenu-item:active .callNodeContextMenuIcon {
+.callNodeContextMenu .react-contextmenu-item--selected .react-contextmenu-icon,
+.callNodeContextMenu .react-contextmenu-item:hover .react-contextmenu-icon {
   filter: invert(1);
 }
 
 .callNodeContextMenuIconMerge {
-  background: url(../../../res/img/svg/merge-icon.svg) center center no-repeat;
+  background-image: url(firefox-profiler-res/img/svg/merge-icon.svg);
 }
 
 .callNodeContextMenuIconCollapse {
-  background: url(../../../res/img/svg/collapse-icon.svg) center center
-    no-repeat;
+  background-image: url(firefox-profiler-res/img/svg/collapse-icon.svg);
 }
 
 .callNodeContextMenuIconFocus {
-  background: url(../../../res/img/svg/focus-icon.svg) center center no-repeat;
+  background-image: url(firefox-profiler-res/img/svg/focus-icon.svg);
 }
 
 .callNodeContextMenuIconDrop {
-  background: url(../../../res/img/svg/drop-icon.svg) center center no-repeat;
+  background-image: url(firefox-profiler-res/img/svg/drop-icon.svg);
```

Comment:
It looks like we are changing some `background-position`s and `background-repeat`s here. Since the initial value of them are `0% 0%` and `repeat`, do you think that can cause a problem?

edit: oh, actually the next commit is about centering the icons. but I think background-repeat question still holds.

Reply:
I factorized this to the class `.react-contextmenu-icon`:
![image](https://user-images.githubusercontent.com/454175/106019232-b4501080-60c2-11eb-9800-c3a4c120b44e.png)
There shouldn't be a difference to what we had before. But I can add a comment mentioning that!

Reply:
Ah, that's right. Yeah, it would be good to mention that :)